mindjunkies/forums/views.py

```
@@ -145,29 +149,91 @@ def get(self, request, *args, **kwargs):
             "reply": reply,
-            "replyForm": reply_form,
+            "replyForm": replyForm,
         }
         return render(request, "forums/reply_form.html", context)
 
     def post(self, request, *args, **kwargs):
 
         reply_id = self.kwargs.get("reply_id")
         reply = get_object_or_404(Reply, id=reply_id)
-        reply_form = ForumReplyForm(request.POST)
+        replyForm = ForumReplyForm(request.POST)
```

Ruff (0.8.2)

Variable `replyForm` in function should be lowercase

(N806)